lib/Runtime/Library/CMakeLists.txt

```diff
@@ -114,6 +114,13 @@ set(CRLIB_SOURCE_CODES
     TypedArray.cpp
     TypedArrayIndexEnumerator.cpp
     UriHelper.cpp
+    WasmLibrary.cpp
```

Comment:
I've added this and Jenkins is green, sending it in in this state as per out-of-band discussion.